Regression/Checksum/benchmarks_json/Python_prev_positions.json

```diff
@@ -1,23 +1,23 @@
 {
+  "lev=0": {
+    "Bx": 0.0,
+    "By": 0.0,
+    "Bz": 0.0,
+    "Ex": 3710588.849989976,
+    "Ey": 0.0,
+    "Ez": 646727.8074440088,
+    "jx": 4745.078379617619,
+    "jy": 368.4331779923921,
+    "jz": 632.1508106460103
+  },
   "electrons": {
     "particle_momentum_x": 8.78764082600202e-23,
     "particle_momentum_y": 3.78128494181519e-24,
-    "particle_momentum_z": 5.396907550474659e-24,
+    "particle_momentum_z": 5.3969075504746586e-24,
```

Comment:
Do you understand why the position was affected?

Reply:
This I would think comes from a difference in the number of digits displayed in the new'ish printing of the updated benchmark values in the Azure output (from which I copied the values to the json files) compared to the digits included in the json writing when the benchmark reset functionality is used.